src/components/tree/Tree.tsx

```diff
@@ -1,79 +1,141 @@
-import React, { FC } from "react";
+import React, { FC, useCallback, useState } from 'react';
+import sortBy from 'lodash.sortby';
 
-import CopyImage from "../vectors/CopyImage";
-import StarImgage from "../vectors/StarImage";
+import CopyImage from '../vectors/CopyImage';
+import StarImgage from '../vectors/StarImage';
+import StarOutline from '../vectors/StarOutline';
 
-import Reference from "./Reference";
-import { Article } from "../../utils/customTypes";
+import Reference from './Reference';
+import { Article } from '../../utils/customTypes';
 
-import "./Tree.css";
-import DATA from "./data.json";
+import './Tree.css';
+import DATA from './data.json';
 
 const INFO: {
   [key: string]: { title: string; info: string };
 } = {
   root: {
-    title: "Root",
+    title: 'Root',
     info: `
       Here you should find seminal articles from the original articles of
       your topic of interest.
     `,
   },
   trunk: {
-    title: "Trunk",
+    title: 'Trunk',
     info: `
       Here you should find articles where your topic of interest got a
       structure, these should be the first authors to discover the
       applicability of your topic of interest
     `,
   },
   leaf: {
-    title: "Leaves",
+    title: 'Leaves',
     info: `
       Here you should find recent articles and reviews that should
       condense very well your topics.
     `,
   },
 };
 
+const DEFAULT_SHOW = {
+  root: true,
+  trunk: true,
+  leaf: true,
+};
+
```

Comment:
Currently, I see that the sections are like radio buttons. I mean, you can just select one of them at the same time. In that way, why don't we use the `DEFAULT_SHOW` variable like a `string` or `null` and show all the sections only if `null` and if it takes a value, then just show the section with that value.